manager/routes/routes.go

```diff
@@ -13,6 +14,7 @@ func Init(app *gin.Engine) {
 }
 
 func InitAPI(group *gin.RouterGroup) {
+	group.Use(middlewares.Authenticator())
```

Comment:
Yes, here it the place I would conditionally put `middlewares.PutTestingIdentity()`.